### DISCUSSION TITLE
Restrict navigating windows under their parent container

### DISCUSSION THREAD

**Discussion by ellyraad** [i3/i3#6136] [Q&A] [answered]
So I have a parent window (red) that has two child windows, and the another window (green) is beside the parent window.

![i3](https://github.com/i3/i3/assets/76833632/d3beea3b-cbe3-4bfb-b416-46419ef7a881)

What I want is that if I am in the 2nd child window in red and I do `focus left`, I want it to cycle back to 1st child, because what's happening is that it goes to green window, since it's on the left of 2nd child.

Is there a way to make the navigation of child windows stays under their parent window?

I hope this makes sense.

**Comment by orestisfl** [comment, marked as answer]
Are you perhaps looking for the `focus_wrapping` setting? https://i3wm.org/docs/userguide.html#_focus_wrapping

**Reply by ellyraad** [reply]
Oh.. thank you. It's

`focus_wrapping force`
